fedmsg/crypto/x509.py

```diff
@@ -196,87 +196,18 @@ def fail(reason):
     signer = subject.get_entries_by_nid(subject.nid['CN'])[0]\
         .get_data().as_text()
 
-    # Perform the authz dance
-    # Do we have a list of permitted senders for the topic of this message?
-    if message.get('topic') in routing_policy:
-        # If so.. is the signer one of those permitted senders?
-        if signer in routing_policy[message['topic']]:
-            # We are good.  The signer of this message is explicitly
-            # whitelisted to send on this topic in our config policy.
-            pass
-        else:
-            # We have a policy for this topic and $homeboy isn't on the list.
-            return fail("Authorization/routing_policy error.  "
-                        "Topic %r.  Signer %r." % (message['topic'], signer))
-    else:
-        # We don't have a policy for this topic.  How we react next for an
-        # underspecified routing_policy is based on a configuration option.
-
-        # Ideally, we are in nitpicky mode.  We leave it disabled while
-        # standing up fedmsg across our environment so that we can build our
-        # policy without having the whole thing come crashing down.
-        if config.get('routing_nitpicky', False):
-            # We *are* in nitpicky mode.  We don't have an entry in the
-            # routing_policy for the topic of this message.. and *nobody*
-            # gets in without a pass.  That means that we fail the message.
-            return fail("Authorization/routing_policy underspecified.")
-        else:
-            # We are *not* in nitpicky mode.  We don't have an entry in the
-            # routing_policy for the topic of this message.. but we don't
-            # really care.  We pass on the message and ultimately return
-            # True later on.
-            pass
-
-    return True
-
-
-def _load_remote_cert(location, cache, cache_expiry, tries=0, **config):
-    """ Get a fresh copy from fp.o/fedmsg/crl.pem if ours is getting stale.
-
-    Return the local filename.
-    """
-    alternative_cache = os.path.expanduser("~/.local" + cache)
-
-    try:
-        modtime = os.stat(cache).st_mtime
-    except OSError:
-        # File does not exist yet.
-        try:
-            # Try alternative location.
-            modtime = os.stat(alternative_cache).st_mtime
-            # It worked!  Use the alternative location
-            cache = alternative_cache
-        except OSError:
-            # Neither file exists
-            modtime = 0
-
-    if (
-        (not modtime and not cache_expiry) or
-        (cache_expiry and time.time() - modtime > cache_expiry)
-    ):
-        try:
-            response = requests.get(location)
```

Comment:
Oh I see - you just moved this code block. I'd still recommend that timeout though… (maybe a separate PR or commit if you prefer - up to you).